go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/knative/client-contrib
```

Comment:
there should be no top-level go.mod but each individual plugin uses its own dependency tree. the same is true for the `vendor/` dir. See the comments to #5 for the reasoning and how I suggest to progress.

Reply:
@rhuss sure, I'm working on it, will move vendor to the admin own structure and resubmit.